src/arviz_stats/sampling_diagnostics.py

```diff
@@ -236,9 +236,6 @@ def rhat(
     * Vehtari et al. (2021).  Rank-normalization, folding, and
       localization: An improved Rhat for assessing convergence of
       MCMC. Bayesian analysis, 16(2):667-718.
```

Comment:
I wonder if we should do something like this to be consistent with the other citations in the rest of the arviz-stats functions:

```suggestion
    .. [1] Vehtari et al. *Rank-normalization, folding, and localization: An improved Rhat 
        for assessing convergence of MCMC*. Bayesian Analysis, 16, 2 (2021). 
        https://doi.org/10.1214/20-BA1221
```